ibm_mq/tox.ini

```diff
@@ -11,7 +11,7 @@ envdir =
     py27: {toxworkdir}/py27
     py38: {toxworkdir}/py38
 description=
-    py{27,38}: e2e ready
+    py{27,38}: e2e ready if __platform != 'Windows'
```

Comment:
Should this be `platform_system` instead of `__platform`? From SQLServer: https://github.com/DataDog/integrations-core/blob/master/sqlserver/tox.ini#L21

Reply:
I guess both work